backfiller/backfiller/main.py

```diff
@@ -24,7 +25,7 @@
 
 
 HOUR_FMT = '%Y-%m-%dT%H'
-TIMEOUT = 5 #default timeout for remote requests 
+TIMEOUT = 5 #default timeout in seconds for remote requests 
 
 def encode_strings(o):
```

Comment:
nitpick: this is also defined in `common` and we should just use that one

Reply:
in fact, this definition is already unused and can just be deleted

Reply:
I don't see in `common` where there is a timeout.

`TIMEOUT` is used as the default value of `timeout` in `list_remote_hours`, `list_remote_segments` and `get_remote_segment`. I suppose I could just replace `timeout` in these with `TIMEOUT`.